tutorials/math/bit-manipulation.md

```diff
@@ -256,6 +274,10 @@ public:
 
 $>>$ shifts the bits to the right. For example $3_{10}$ ($0011_2$) $>> 1$ would become $1$ ($0010_2$).
 
+$4 >> 1$ dividing $4$ by $2$. Also $n >> m$ means dividing $n$ by $2$ power m. i.e, $n = n / (2^m)$
+
+In simple, $n >> m$ _shifting each bit of n to right m times_. Let's say, $n = 8$ and $m = 2$. $n$ can be represented as $1000_2$ in binary, Therefore, $8 >> 2$ = $1000_2$ << $2$$ = $0010_2$ (4), which is same as $(8 / 2^2)$ = $4$
```

Comment:
`$8$ >> $2$ = $1000_2$ >> $2$ = $0010_2$ (4)`

Reply:
Wrong with the syntax, will change it. 

Since the given example, was just only handled Right shift (>>) operator with 1. 

What if we decide to right shift more than once, 2 times or 3 times, this is to calculate the number with the given formula.